src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -35,7 +45,35 @@
      * Андрей - 600
      * София - 900
      */
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder("Отчёт за период " + dateFrom + " - "
+                + dateTo + "\n");
+        SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
+        try {
+            Date from = format.parse(dateFrom);
+            Date to = format.parse(dateTo);
+            if (to.before(from)) {
+                return null;
+            }
+            int[] totalSalaries = new int[names.length];
+            for (int i = 0; i < names.length; i++) {
+                for (String s : data) {
```

Comment:
`s` isn't very understandable name

Reply:
can't I use such type of variable name in for each loop, similar to "i" variable name in for loop?